boards/nxp/frdm_mcxn947/frdm_mcxn947-pinctrl.dtsi

```diff
@@ -37,4 +37,31 @@
 			bias-pull-up;
 		};
 	};
+
+	pinmux_enet_qos: pinmux_enet_qos {
+		mdio_group {
```

Comment:
I'm not sure there is a specific requirement here, but we typically group pins based on shared configuration rather than settings (so the RX and TX group could be combined). Only thing I can find in pinctrl documentation is:
> One of the advantages of this choice is the grouping capability based on shared pin configuration

So I'll leave the decision up to you on if these should be grouped or not

Reply:
ok